### PR TITLE
update content organization to format for Cell genomics

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose

- reorder and add star method stubs per https://els-jbs-prod-cdn.jbs.elsevierhealth.com/pb-assets/journals/research/cell/methods/Methods_Guide_Cell-1622124325393.pdf
- start acknowledgements, add contact info, etc

### Issue

NA

#### Which areas should receive a particularly close look?
major updates were the STAR METHODS sections added - per the above link, the language is very specific, so this probably won't be modified too much.

Are there anymore acknowledgements to add at this point? Else, we can add an HTML tag to revisit, but just wanted to get it in there for now.

@jaclyn-taroni - OK with me being lead contact for the data?

#### Is there anything that you want to discuss further?
- still need to add P30 funding (awaiting grant number) - completed
- need to add pedcbio link once @migbro creates study per https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1185 - completed
- need to create and add DOIs for datasets - will do as a separate issue since this has yet to be done (#175)

#### Is the pull request ready for review?
No

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [x] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
I would wait until everything that is currently open goes in to renumber, etc.

File: content/05.administrative.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+## Acknowledgments
+
+We graciously thank the patients and families who have donated their tumors to the Children's Brain Tumor Network and/or the Pacific Pediatric Neuro-oncology Consortium, without which, this research would not be possible. 
+This work was funded through the Alex’s Lemonade Stand Foundation (ALSF) Childhood Cancer Data Lab (JNT, CSG, JAP, CLS, CJB), ALSF Young Investigator Award (JLR), ALSF Catalyst Award (JLR, ACR, PBS), NCI/Leidos contract 18X104F5 (JLR, ACR, APH), Children’s Hospital of Philadelphia Division of Neurosurgery (PBS and ACR), the Australian Government, Department of Education (APH), and NIH Grants 3P30 CA016520-44S5 (ACR), U2C HL138346-03 (ACR, APH), and U24 CA220457-03 (ACR).
```
Comment (jaclyn-taroni) [inline content/05.administrative.md:4]:
```suggestion
This work was funded through the Alex’s Lemonade Stand Foundation (ALSF) Childhood Cancer Data Lab (JNT, CSG, JAS, CLS, CJB), ALSF Young Investigator Award (JLR), ALSF Catalyst Award (JLR, ACR, PBS), NCI/Leidos contract 18X104F5 (JLR, ACR, APH), Children’s Hospital of Philadelphia Division of Neurosurgery (PBS and ACR), the Australian Government, Department of Education (APH), and NIH Grants 3P30 CA016520-44S5 (ACR), U2C HL138346-03 (ACR, APH), and U24 CA220457-03 (ACR).
```

File: content/05.administrative.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+|Author|Contributions|
+|---|---|{% for author in manubot.authors %}
+|{{author.name}}|{% for contribution in author.contributions %}{{ contribution }}{% if not loop.last %}, {% endif %}{% endfor %}|{% endfor %}
+
+
+## Declarations of Interest
+
+CSG's spouse is an employee of Alex's Lemonade Stand Foundation, which was a sponsor of this research.
+
```
Comment (jaclyn-taroni) [inline content/05.administrative.md:19]: This probably makes sense then:

```suggestion
JAS, CLS, CJB, and JNT are or were employees of Alex's Lemonade Stand Foundation, a sponsor of this research.
```

File: content/05.administrative.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+## Author Contributions
+
+|Author|Contributions|
+|---|---|{% for author in manubot.authors %}
+|{{author.name}}|{% for contribution in author.contributions %}{{ contribution }}{% if not loop.last %}, {% endif %}{% endfor %}|{% endfor %}
+
+
+## Declarations of Interest
+
+CSG's spouse is an employee of Alex's Lemonade Stand Foundation, which was a sponsor of this research.
```
Comment (jaclyn-taroni) [inline content/05.administrative.md:18]:
```suggestion
CSG's spouse was an employee of Alex's Lemonade Stand Foundation, which was a sponsor of this research.
```

**Review by jaclyn-taroni — COMMENTED** [review]
I'm not going to approve yet because I expect some more changes, but wanted to provide some comments.

**Comment by jharenza** [comment]
@jaclyn-taroni-  this is ready for re-review.

@cgreene - I swapped my and your authorship orders after discussion with @jaclyn-taroni and just wanted to let you know/make sure that is OK with you.

**Review by jaclyn-taroni — APPROVED** [review]
This approval is conditional on @cgreene's approval

**Comment by cgreene** [comment]
Yup - this is fine!

**Review by cgreene — APPROVED** [review]

**Pull request merged**